### PR TITLE
feat(monitoring): add logs for monitoring

### PR DESCRIPTION
**Pull request by RoeiRom**
# The Monitoring 🖥
This MR adds monitoring to the service
## The Problem
We have a container that prints to the console but the output is not informative and not being collected by other service for analytics
## The solution
Add informative logs using [Winston](https://www.npmjs.com/package/winston) and they will be collected by [Azure Application Insights](https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview)

### CODE REVIEW

**Review by MayaSasson — APPROVED** [review]

File: src/DAL/index.ts [hunk 1]
```diff
@@ -15,15 +18,28 @@ const pool = new Pool({
 });
 
 pool.on('connect', () => {
-    console.log('initiated db connection pool');
+    logger.info({
```
Comment (EmilyLip) [inline src/DAL/index.ts:21]: where is the workflow?

File: src/DAL/index.ts [hunk 2]
```diff
@@ -15,15 +18,28 @@ const pool = new Pool({
...
+        });
         await pool.query(deleteInvestigationsQuery(subDays(new Date(), +process.env.MIN_DAYS_TO_REMOVE), countiesBlackList));
-        console.log('finished deleting investigations successfully');
+        logger.info({
+            step: 'finished deleting investigations successfully',
+            severity: Severity.LOW
+        });
     } catch (error) {
+        logger.error({
+            step: 'got error in querying the DB, throwing an exception',
```
Comment (EmilyLip) [inline src/DAL/index.ts]: add the error to the log

File: src/DAL/index.ts (hunk 1 above)
Comment (RoeiRom) [reply src/DAL/index.ts:21]: in the created log there is the workflow

File: src/DAL/index.ts (hunk 2 above)
Comment (RoeiRom) [reply src/DAL/index.ts]: the error is being printed afterwards in the catch block that catches the exception

Comment (EmilyLip) [reply src/DAL/index.ts]: i think one log about the error is enoughl, combinw them or leave only one of them

File: src/DAL/index.ts (hunk 1 above)
Comment (RoeiRom) [reply src/DAL/index.ts:21]: because there is only one flow in this service

File: src/DAL/index.ts (hunk 2 above)
Comment (RoeiRom) [reply src/DAL/index.ts]: Done, removed this log

**Review by EmilyLip — APPROVED** [review]

**Review by doronlnd84 — APPROVED** [review]

**Pull request merged**